Listenbrainz/UI/Components/SettingsView.swift

```diff
@@ -10,10 +10,11 @@ import SwiftUI
 
 struct SettingsView: View {
   @AppStorage("isDarkMode") private var isDarkMode = false
-  @AppStorage("userToken") private var userToken:String = ""
+  @State  var userToken: String
```

Comment:
The app is not working because of this. Revert this

Reply:
actually it was working in my side, but now I have kept that in App Storage